ui/lib/core/addon/components/get-credentials-card.js

```diff
@@ -43,8 +45,11 @@ export default class GetCredentialsCard extends Component {
     if (role) {
       this.router.transitionTo('vault.cluster.secrets.backend.credentials', role);
     }
+    // TODO kv engine cleanup. Should be able to remove the is-engine conditional.
```

Comment:
I created an `OverviewCard` component - it's a more generalized component and also uses the new HDS Card component. Since we're trying to move towards using components that let the component itself handle their respective logic, the `OverviewCard` might be good to use here. An example can be found [here](https://github.com/hashicorp/vault/blob/a4f67a6b2b3b5ffefb2c193f166b6b10cca541bb/ui/lib/kubernetes/addon/components/page/overview.hbs#L20)!

Reply:
Awesome, thank you so much for pointing that out. Amending now.